mfs.go

```diff
@@ -0,0 +1,370 @@
+package shell
+
+import (
+	"context"
+	"io"
+
+	files "github.com/ipfs/go-ipfs-files"
+)
+
+type MfsLsEntry struct {
+	Name string
+	Type uint8
+	Size uint64
+	Hash string
+}
+
+type filesLsOutput struct {
+	Entries []*MfsLsEntry
+}
+
+type filesFlushOutput struct {
+	Cid string
+}
+
+type filesStatOutput struct {
+	Blocks         int
+	CumulativeSize uint64
+	Hash           string
+	Local          bool
+	Size           uint64
+	SizeLocal      uint64
+	Type           string
+	WithLocality   bool
+}
+
+type filesOpt func(*RequestBuilder) error
+type filesLs struct{}
+type filesChcid struct{}
+type filesMkdir struct{}
+type filesRead struct{}
+type filesWrite struct{}
+type filesStat struct{}
+
+var (
+	FilesLs    filesLs
+	FilesChcid filesChcid
+	FilesMkdir filesMkdir
+	FilesRead  filesRead
+	FilesWrite filesWrite
+	FilesStat  filesStat
+)
+
+// Long use long listing format
```

Comment:
Hm. Maybe `Stat`? Long refers to the text format. It doesn't really make sense in a library.

Reply:
OK，I'll rename it.